install_gh.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+#################################
+# Installs github cli tool
+# Arguments:
+# None
+# Author: github.com/sosaeio
+# Date: Oct 28, 2023
+#################################
+
+GREEN='\033[0;32m'
+YELLOW='\033[0;33m'
+NC='\033[0m' # No Color
+CM=$(perl -e 'print "\xE2\x9C\x94"') #checkmark
+CURRDIR=$PWD
+
+# OS Specific Information
+OSTYPE=$(uname -s)
+GOFILE="go1.21.3.darwin-arm64.pkg"
+
+# Check if the gh tool has been installed
+check_gh=$({ which gh | grep -o gh > /dev/null && echo 1 || echo 0; })
+
+#Check if go is installed
+check_go=$(go version | grep "go version" -o > /dev/null && echo 1 || echo 0)
+
+function install_go(){
```

Comment:
why are you using functions?

Reply:
This was to check if go was already installed in the Mac. Since it's needed to build gh from source. I'm removing the condition and installing it regardless.